### DISCUSSION TITLE
Skylight Emitter

### DISCUSSION THREAD

**Discussion by Gzhji** [mitsuba-renderer/mitsuba3#962] [Q&A]
Dear community,

I was wondering whether the Skylight Emitter has been removed from Mitsuba3.
This emitter used to be in Mitsuba0.6.

Thanks in advance,
Guanzhou

**Comment by njroussel** [comment]
Hi @Gzhji 

Indeed, the `skylight` Emitter from MItsuba 0.6 is not available in Mitsuba 3. It has not been ported (yet).

We don't have anyone actively working on it and don't plan to work on it anytime soon. I'd welcome a PR from anyone that wants to port this feature to the latest version :smile:
